composer.json

```diff
@@ -73,6 +73,12 @@
                 "Issue #2922677: Uncaught TypeError: Cannot read property 'replace' of undefined": "https://www.drupal.org/files/issues/2023-02-15/2922677-33.patch",
                 "https://dgo.re/3049332 LB Call to a member function getEntityTypeId() on null": "https://www.drupal.org/files/issues/2024-03-11/drupal-core--2024-01-09--3049332-87.patch",
                 "https://dgo.to/3344041 Allow textarea widgets to be used for text (formatted) fields (and check maxlength)": "https://www.drupal.org/files/issues/2023-02-23/3344041-text-formatted-field-use-textarea-widget.patch"
+            },
+            "drupal/inline_entity_form": {
+                "https://dgo.to/3002175 Custom blocks created in inline forms should not be reusable": "https://www.drupal.org/files/issues/2024-04-18/3002175-2.x-47.patch"
+            },
+            "drupal/entity_clone": {
+                "https://dgo.to/3050027 Inline Blocks on Layout Builder Fail to Clone Correctly": "https://www.drupal.org/files/issues/2023-08-14/3050027-62.patch"
```

Comment:
also feels a risky one. 
https://www.drupal.org/project/entity_clone/releases/2.1.0-beta1 was tagged, but patch wasn't included inside... @froboy

Reply:
https://dgo.to/3050027 was removed due to a regression, and it's also related to https://www.drupal.org/project/entity_clone/issues/3258535. I agree we should try to push some of the other issues forward before merging this for everyone. I have tested on a client project and it's worked successfully.

Reply:
Updated this to apply to [2.1.0-beta1](https://www.drupal.org/project/entity_clone/releases/2.1.0-beta1)